src/Beacon.jsx

```diff
@@ -157,6 +169,13 @@ export class Beacon extends Component {
         this.setState({ targetElement, targetBounds }); // eslint-disable-line react/no-did-update-set-state
       }
     }
+    if (this.state.tooltip && !this.state.tooltipBounds) {
+      // Save tooltip element bounds when tooltip is active
+      this.setState({
+        tooltipBounds: this.refs.tooltip.getBoundingClientRect(),
+        arrowBounds: this.refs.tooltipArrow.getBoundingClientRect()
+      });
```

Comment:
Saving DOM values for tooltip and arrow to work with the sizing before CSS animation begins.